### ISSUE TITLE
[BUG] - Unable to install via Community Plugins

### ISSUE THREAD

**Issue by SoRobby** [st3v3nmw/obsidian-spaced-repetition#534]
**Describe the bug**
When attempting to install Spaced Repetition through Obsidians Community Plugin view it keeps failing. I've attempting to install the plugin with a clean vault (no other plugins or themes) and it still fails. Two days ago I was able to install the plugin, now it keeps failing to install (even in new vaults).

**To Reproduce**
Attempt to install the plugin through Obsidians Community Plugin view. Did not try to install manually.

**Screenshots**
![image](https://user-images.githubusercontent.com/16641866/204163150-b9f16c9b-50d1-47dc-9402-81d99bdd3b9f.png)

**Versions (please complete the following information):**

-   OS: Win11
-   Obsidian version: v1.0.3
-   Plugin version: v1.9.0
-   If on desktop, Installer version: '5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) obsidian/0.15.8 Chrome/100.0.4896.160 Electron/18.3.5 Safari/537.36'

**Comment by Faeranne** [comment]
Looks like the latest release hasn't been finished yet.  The manifest lists version 1.9.0, but the latest release is still 1.8.0.  I'm also not seeing a 1.9.0 tag yet, so that may be while the release isn't available just yet.

**Comment by j-r-e-i-d** [comment]
same happening here

**Comment by rc14193** [comment]
I'm having the same issue. Doing a manual install worked and I was able to activate from there but can't install from the gui. It also lists it as ready to update and fails though.

**Comment by saeneas** [comment]
same here

**Comment by mdesevricourt** [comment]
same

**Comment by op1490** [comment]
Same

**Comment by daxmonson** [comment]
+1

**Comment by aldrinjenson** [comment]
same issue. Please add a fix soon🤞

**Comment by guxiyuesi** [comment]
same

**Comment by sahmoli** [comment]
same issue.

**Comment by outbymine** [comment]
same

**Comment by valiantgenomics** [comment]
Same issue here with Obsidian version 1.0.3 on Linux. Cannot update the plugin. Tried uninstalling it so I can reinstall it, and now can't even install it again.

**Comment by grantx2016** [comment]
same

**Comment by alanalvarado** [comment]
Uninstalled it, Reinstalled it, and now it won't install.

**Comment by NeilClack** [comment]
Guys, and anyone from the future who sees this
The issue has been clearly defined above. 
The manifest is listing a version that has not yet been released. 
I'm sure it's coming soon, but until then, install manually or wait. 
There is no point in repeatedly posting "same here on X", especially when it's not an OS related issue. 

The manual installation instructions can be found on the plugin page, or in the README on the homepage of this repository.

**Comment by gork3n** [comment]
I've tried manual installation and the plugin still didn't work. I'm going to try again right now. I'll edit this post to let you all know.

**Comment by AB1908** [comment]
Hi all. Terribly sorry for breaking this in #531. Will add an interim release from my fork. Sorry for the trouble.

**Comment by SunnyOd** [comment]
Having this issue with version 1.9.0 of spaced repetition and obsidian 1.1.3

**Comment by gregheth** [comment]
I cannot install - macos 10.13.6

Can we install an old version?
What is the estimated time to fix?
Should I use Anki instead?

**Comment by ollecello** [comment]
same

**Comment by AB1908** [comment]
Do my install instructions via brat not help?

**Comment by grantx2016** [comment]
> Do my install instructions via brat not help?

This blocking bug has been around 2 weeks which is long enough that it should be just fixed without expecting end users to implement workarounds. Just my opinion.

**Comment by AB1908** [comment]
I agree but the author seems to be busy and I'm only a volunteer trying to help people here.

**Comment by grantx2016** [comment]
> I agree but the author seems to be busy and I'm only a volunteer trying to help people here. I'm not really obligated to help anyone but I hang around since I really like this plugin.

Of course. I appreciate you are trying to help and for some it will be helpful. 👍 

I also understand this is the nature of open source and the author is not getting paid for this work but I also think some commitment is required of the authors to fix blocking bugs quickly or it undermines the confidence in Obsidian as a whole.

**Comment by SunnyOd** [comment]
@grantx2016 that's a bit dramatic imo. Life gets in the way sometimes and it's Christmas! I'm sure everybody understands

**Comment by grantx2016** [comment]
> dramatic

I'd call it realistic. End users don't care about the life challenges of open source devs. Cold but true. If Obsidian publishes a plug-in I don't think it's unreasonable for end users, who build their workflow around it, to expect that it be at least maintained operable. 2+ weeks for a fixing a blocking bug is too long, imo. If Obsidian don't start setting some plug-in standards along these lines then mass take up will be impeded and that hurts all the responsible devs who keep their code in order.

**Comment by AB1908** [comment]
>  If Obsidian publishes a plug-in I don't think it's unreasonable for end users

While I realize we are bikeshedding here, can you explain what you mean by "Obsidian publishes a plugin"? 

> If Obsidian don't start setting some plug-in standards along these lines then mass take up will be impeded and that hurts all the responsible devs who keep their code in order.

The Obsidian team is fairly transparent when they say that they are not responsible for ensuring community plugins continue to function properly after their initial review and listing on the community store.

All this said, I apologize for causing a breakage and have attempted to provide a reasonable workaround to help you get your work done meanwhile. Unsure what else you're asking for here.

**Comment by grantx2016** [comment]
> > If Obsidian publishes a plug-in I don't think it's unreasonable for end users
> 
> While I realize we are bikeshedding here, can you explain what you mean by "Obsidian publishes a plugin"?
> 
> > If Obsidian don't start setting some plug-in standards along these lines then mass take up will be impeded and that hurts all the responsible devs who keep their code in order.
> 
> The Obsidian team is fairly transparent when they say that they are not responsible for ensuring community plugins continue to function properly after their initial review and listing on the community store.
> 
> All this said, I apologize for causing a breakage and have attempted to provide a reasonable workaround to help you get your work done meanwhile. Unsure what else you're asking for here.

Try not to be offended by my comments @AB1908. That's not my intent. I don't expect anything from you. Like others, I would just like this blocking bug fixed, properly, as there has been ample time to do so. I'm just trying to point out that end users don't care for all this dev stuff. You are working from a belief that Obsidian users are devs who have GitHub accounts, dev environments ets. Obsidian is a note taking app, not a sandpit for devs to tinker around on, jor at least it shouldn't be, imo. Anyway, clearly we have different perspectives on how plugins should be managed so let's just leave it here as this is obviously now way off topic and one better to raise in the Obsidian forums. I appreciate you have tried to help but I think this is one of the author.

**Comment by grantx2016** [comment]
> I apologize for causing a breakage and have attempted to provide a reasonable workaround to help you get your work done meanwhile. Unsure what else you're asking for here.

Try not to be offended by my comments @AB1908. That's not my intent. I don't expect anything from you. Like others, I would just like this blocking bug fixed, properly, as there has been ample time to do so. I'm just trying to point out that end users don't care for all this dev stuff. You are working from a belief that Obsidian users are devs who have GitHub accounts, dev environments ets. Obsidian is a note taking app, not a sandpit for devs to tinker around on, jor at least it shouldn't be, imo. Anyway, clearly we have different perspectives on how plugins should be managed so let's just leave it here as this is obviously now way off topic and one better to raise in the Obsidian forums. I appreciate you have tried to help but I think this is one of the author.

**Comment by hicaicai** [comment]
> without expecting end users to implement workarounds

I do understand your emotions, but this could be a longer wait, what will you do if they don't update anymore?

**Comment by st3v3nmw** [comment]
Hey guys, doing a release later today.

Sorry for the inconvenience, been a little busy.

And thanks @AB1908 for the help!

**Comment by AB1908** [comment]
Glad to help. More than happy if you'd like to add me as a collaborator so I can smooth over any patches if you can't find the time. FWIW, the note review PR is somewhat broken and I'll need to patch it.

**Comment by valiantgenomics** [comment]
> Hey guys, doing a release later today.
> 
> Sorry for the inconvenience, been a little busy.
> 
> And thanks @AB1908 for the help!

No problems man, you don't have to apologise for anything. The fact that you're developing and maintaining this awesome plugin in the first place is already outstanding. Thanks a ton for letting us know that a fix is coming soon!

**Comment by mayukhc** [comment]
> Hey guys, doing a release later today.
> 
> Sorry for the inconvenience, been a little busy.
> 
> And thanks @AB1908 for the help!

This plugin really helped me study, thanks for making this @st3v3nmw and can't wait for 1.9. Also, this bug could be easily bypassed via manual install, so was not a blocker to anybody imo.

**Comment by st3v3nmw** [comment]
Version 1.9.0 is out!

https://github.com/st3v3nmw/obsidian-spaced-repetition/releases/tag/1.9.0
